app/src/main/java/com/example/perfectweatherallyear/MainActivity.kt

```diff
@@ -21,8 +20,12 @@ class MainActivity : AppCompatActivity() {
     @RequiresApi(Build.VERSION_CODES.O)
     override fun onStop() {
         super.onStop()
-        NotificationHandler.createNotificationChannel(this.application)
+        startService()
```

Comment:
It is not cleared from the name what service will be started.
 [Check this guide](https://habr.com/ru/post/424051/)